README.md

```diff
@@ -213,14 +213,15 @@ ENCRYPTION OPTIONS:
 POWER TOYS:
       --ai                 auto-import from Firefox/Chrome/Chromium
       -e, --export file    export bookmarks to Firefox format HTML
+                           export XBEL, if file ends with '.xbel'
```

Comment:
Please add at the end. Why is this turning up to be the second option?

Reply:
I just thought it was more related to the HTML format so to keep things tidy I put it there next to HTML as Org and MD are kept next to each other.
But to keep things as they are I will add to the end.

Reply:
In that case. the current order is fine.

Reply:
So I will revert the last change, Thanks.